src/components/ProgressBar.scss

```diff
@@ -0,0 +1,82 @@
+// @import "../base/index.scss";
+
+//------------------------------------------------------------//
+
+// * Progress Bar
+
+// TODO : "completed" status originally appeared in both the
+//        positive group and the neutral group
+
+//------------------------------------------------------------//
+
+.ProgressBar {
+}
+
+.ProgressBar-title {
+  margin: 0;
+  padding-top: $su-small;
+  padding-bottom: $su-xsmall;
+  text-align: left;
+  .ProgressBar--small & {
+    font-size: $tu-small-fontSize;
+  }
+  .ProgressBar--large & {
+    font-size: $tu-large-fontSize;
+  }
+}
+
+.ProgressBar-range {
+  position: relative;
+  background: $cu-background;
+  height: $su-small;
+  border-radius: ($su-small / 2);
+  margin-top: $su-xsmall;
+  margin-bottom: $su-default;
+  box-shadow: $inset-box-shadow-medium;
+  overflow: hidden;
+  clear: both;
+  .ProgressBar--small & {
+    height: $su-xsmall;
+    border-radius: ($su-xsmall / 2);
+    box-shadow: $inset-box-shadow-small;
+  }
+  .ProgressBar--large & {
+    height: $su-default;
+    border-radius: ($su-default / 2);
+  }
+}
+
+.ProgressBar-status {
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: $cu-success;
+  // gets overridden inline
+  width: auto;
+}
+
+//----------------------------------------------//
+// Styles the progress bar for different status
+//---------------------------------------------//
+
+.ProgressBar-status--active,
+.ProgressBar-status--success,
+.ProgressBar-status--waitingForPayment,
+.ProgressBar-status--paid,
+.ProgressBar-status--completed {
+  background: $cu-positive;
+}
```

Comment:
My question would be, are there any cases where something other than the color of the progress bar relies on distinguishing between different cases? If not, perhaps what's in Classic is covering a variety of legacy states that aren't needed for this updated version of the component.